/**:
  ros__parameters:
    x_pid:
      antiwindup: True
      p: 5.0
      i: 1.5
      d: 0.3
      i_clamp_max:  0.05
      i_clamp_min: -0.05
    y_pid:
      antiwindup: True
      p: 5.0
      i: 1.5
      d: 0.3
      i_clamp_max:  0.05
      i_clamp_min: -0.05
    w_pid:
      antiwindup: True
      p: 5.0
      i: 1.5
      d: 0.3
      i_clamp_max:  0.05
      i_clamp_min: -0.05
    x_pid_icp:
      antiwindup: True
      p: 1.5
      i: 0.5
      d: 0.0
      i_clamp_max:  0.05
      i_clamp_min: -0.05
    y_pid_icp:
      antiwindup: True
      p: 1.5
      i: 0.5
      d: 0.0
      i_clamp_max:  0.05
      i_clamp_min: -0.05
    w_pid_icp:
      antiwindup: True
      p: 1.5
      i: 0.5
      d: 0.0
      i_clamp_max:  0.05
      i_clamp_min: -0.05
